integration/lighthouse.ts

```diff
@@ -0,0 +1,20 @@
+import {SecretKey} from "@chainsafe/bls";
+import {Keystore} from "@chainsafe/bls-keystore";
+import keystore from "./lighthouse-keystore.json";
+import assert from "assert";
+
+import {restValidation} from "./restValidation";
+const sk = "222222222222222222222222222222222222222222222222222";
```

Comment:
password?

Reply:
its imported validator Keystore generated by lighthouse which contains secret key like this

Reply:
from what I see in code you are decrypting secret key from keystore and use this variable as password for decryption

Reply:
`sk` means secret key

Reply:
My point is that variable name should be password haha